sections/05_treat.md

```diff
@@ -375,15 +375,9 @@ atom-atom interactions and the forces that govern binding affinity.
 
 #### *De novo* drug design
 
-Whereas the goal of virtual screening is to find active molecules by predicting
```

Comment:
`doi:10.1021/acs.jmedchem.5b01849` was a good reference so I prefer to not cut it.  I'm okay with the reduced text.

@mrwns wrote some of this as well.